openmc/arithmetic.py

```diff
@@ -532,23 +474,6 @@ def __eq__(self, other):
     def __ne__(self, other):
         return not self == other
 
-    def __deepcopy__(self, memo):
-        existing = memo.get(id(self))
```

Comment:
I'm curious - is this line included in the default implementation? Seeing this is now sparking my memory of *one* of the reasons I went about implementing `__deepcopy__` methods back in the day - to boost performance by eliminating redundant copies of objects in CSG trees (*i.e.*, this line of code). I want to say that this is not included in the default implementation, but I'm not sure. Regardless, I don't *think* it matters for the `Cross*` and `Aggregate*` classes anyway.